src/lib/components/UrlChooser/index.js

```diff
@@ -0,0 +1,185 @@
+import {html} from "lit-element";
+import {store} from "../../store";
+import {BaseElement} from "../BaseElement";
+import {userRef, firebase} from "../../fb";
+
+/* eslint-disable require-jsdoc */
```

Comment:
Would you mind adding a fileoverview to help explain what this class does?

Reply:
Done